package-lock.json

```diff
@@ -1887,6 +1887,12 @@
             "integrity": "sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=",
             "dev": true
         },
+        "nan": {
```

Comment:
Are these non-sleep changes supposed to be here?  What are they for?

Reply:
I'm not sure, that was automatically added from `npm install`!

Reply:
Looks like nan is a dependency: https://www.npmjs.com/package/sleep.  `string_decoder` looks like it's the same and just got moved elsewhere in the file, so these changes should be ok.